WalletWasabi/KeyManagement/HdPubKey.cs

```diff
@@ -18,27 +19,14 @@ public class HdPubKey : IEquatable<HdPubKey>
 		[JsonConverter(typeof(KeyPathJsonConverter))]
 		public KeyPath FullKeyPath { get; }
 
-		private string _label;
-
 		[JsonProperty(Order = 3)]
-		public string Label
-		{
-			get => _label;
-			private set
-			{
-				value = Guard.Correct(value);
-				if (value != _label)
-				{
-					_label = value;
-					HasLabel = !string.IsNullOrEmpty(value);
-				}
-			}
-		}
+		[JsonConverter(typeof(LabelJsonConverter))]
```

Comment:
```suggestion
		[JsonConverter(typeof(SmartLabelJsonConverter))]
```

Reply:
Thanks, I did it manually instead of accepting your commit suggestions, to make sure I won't miss out anything.